.expeditor/scripts/shared.sh

```diff
@@ -272,3 +279,23 @@ fastly_token() {
           -field=token \
           account/static/fastly/eng-services-ops
 }
+
+latest_release_tag() {
+  local repo="${1?repo argument required}"
+  tag=$(curl --silent "https://api.github.com/repos/${repo}/releases/latest" | jq -r .tag_name)
+  echo "${tag}"
+}
+
+install_hub() {
+  # TODO: Create a Hab core plans pkg for this.
```

Comment:
It'd be nice to create an issue for this and add a link to it here in the comments (also, link that issue to this PR :wink:)

Reply:
Handled in https://github.com/habitat-sh/habitat/pull/7258/commits/82903e6708081b469ceb334c1ea88778d0d5d4b1